explorer/src/components/TestnetRewards/TestnetRewardsTable.tsx

```diff
@@ -172,93 +200,101 @@ export const TestnetRewardsTable: FC = () => {
   const cleanCell = (cell: string) => cell.replace(/"/g, '').replace(/,/g, '')
 
   const handleLoad = useCallback(async () => {
-    const file = await fetch('/data/testnet/astralTestnetRewardsMainnetAllocations-10oct2024.csv')
+    const file = await fetch('/data/testnet/astralTestnetRewardsMainnetAllocations-10oct2024-2.csv')
```

Comment:
Why is this file `-2` instead of substituting?

Reply:
Will substitute them on the next version (still waiting for final confirmation on Aries allocation)